<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-10</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>850</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="SensorInformation">
     <attribute name="title">
      <string>SensorInformation</string>
     </attribute>
     <attribute name="toolTip">
      <string>Click to see status of sensors</string>
     </attribute>
     <attribute name="whatsThis">
      <string>Shows status of sensors</string>
     </attribute>
     <widget class="QProgressBar" name="FreshWaterProgressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>270</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adjust to the content of the fresh water tank&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="BlackWaterProgressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>270</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="GreyWaterProgressBar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>270</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="BatteryProgressBar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>300</y>
        <width>270</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="BlackWaterLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Black Water&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="FreshWaterLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Fresh Water&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="GreyWaterLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Grey Water&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="BatteryLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Battery&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1250</width>
        <height>850</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/background.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>StatusBackground</zorder>
     <zorder>FreshWaterProgressBar</zorder>
     <zorder>BlackWaterProgressBar</zorder>
     <zorder>GreyWaterProgressBar</zorder>
     <zorder>BatteryProgressBar</zorder>
     <zorder>BlackWaterLabel</zorder>
     <zorder>FreshWaterLabel</zorder>
     <zorder>GreyWaterLabel</zorder>
     <zorder>BatteryLabel</zorder>
    </widget>
    <widget class="QWidget" name="UsageHistory">
     <attribute name="title">
      <string>UsageHistory</string>
     </attribute>
     <attribute name="toolTip">
      <string>Click to see the history of the devices usage through graphs</string>
     </attribute>
     <attribute name="whatsThis">
      <string>Shows the history of the device's usage through graphs</string>
     </attribute>
     <widget class="QLabel" name="HistoryBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/background.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckGraphs">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>87</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="MplWidget" name="SensorGraph" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>379</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Power">
     <attribute name="title">
      <string>Power</string>
     </attribute>
     <widget class="QLabel" name="PowerBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/background.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="powerApp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>195</width>
        <height>205</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>195</width>
        <height>205</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Click to turn off the Application</string>
      </property>
      <property name="whatsThis">
       <string>Turns off the application</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/powerApp.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="powerRasberry">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>195</width>
        <height>205</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>195</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Click to turn off the Raspberry Pi</string>
      </property>
      <property name="whatsThis">
       <string>Turns off the Raspberry Pi</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/raspberrypi.png);</string>
      </property>
      <property name="text">
       <string>Turn off Pi</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>720</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>850</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/background.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>TabWidget</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>powerApp</tabstop>
  <tabstop>TabWidget</tabstop>
  <tabstop>powerRasberry</tabstop>
  <tabstop>CheckGraphs</tabstop>
 </tabstops>
 <resources>
  <include location="../new.qrc"/>
 </resources>
 <connections/>
</ui>
